admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +161,133 @@ A failure from this check indicates that a significant amount of space in etcd
 is being taken up by {product-title} image data, which can eventually result in
 your etcd cluster crashing.
 
-A user-defined limit may be set by passing the variable
-`etcd_max_image_data_size_bytes=400000000` to the `openshift_health_checker`
-role.
+A user-defined limit may be set by passing the `etcd_max_image_data_size_bytes`
+variable. For example, setting `etcd_max_image_data_size_bytes=40000000000` will
+cause the check to fail if the total size of image data stored in etcd
+exceeds 40 GB.
+
+|`etcd_traffic`
+|This check detects higher than normal traffic on an etcd host. It fails if a
+`journalctl` log entry with an etcd sync duration warning is found.
+
+For further information on improving etcd performance, see
+xref:host_practices.adoc#scaling-performance-capacity-host-practices[Recommended Host Practices].
```

Comment:
I think this meant to link more specifically to the `scaling-performance-capacity-host-practices-etcd` section. Updating the link to that and also adding a link to the KBase.